cmd/cmdoptions/scenariorunner.go

```diff
@@ -0,0 +1,117 @@
+package cmdoptions
```

Comment:
I think the existing `loadgen` package should be the package used programmatically, we don't need a whole new package